app/ts/components/requirements.tsx

```diff
@@ -94,11 +94,36 @@ export const Immutable = ( { checkBoxesArray } : { checkBoxesArray: OptionalSign
 	if (checkBoxesArray.deepValue === undefined) return <></>
 	const checkBoxes = checkBoxesArray.deepValue[checkBoxesArray.deepValue.length - 1]
 	if (checkBoxes === undefined || checkBoxes.type !== 'finalChild') return <></>
+
+	function humanReadableDateDelta(secondsDiff: number) {
+		const secondsInHour = 3600
+		const secondsInDay = secondsInHour * 24
+		const secondsInMonth = secondsInDay * 30
+		const secondsInYear = secondsInDay * 365
+
+		const years = Math.floor(secondsDiff / secondsInYear)
+		secondsDiff %= secondsInYear
+
+		const months = Math.floor(secondsDiff / secondsInMonth)
+		secondsDiff %= secondsInMonth
+
+		const days = Math.floor(secondsDiff / secondsInDay)
+
+		const parts: string[] = []
+		if (years > 0) parts.push(`${ years } years`)
+		if (months > 0) parts.push(`${ months } months`)
+		if (days > 0) parts.push(`${ days } days`)
+
+		return parts.length > 0 ? parts.join(' ') : 'less than a day'
+	}
```

Comment:
This function doesn't account for leap years or leap minutes.  Probably close enough though.